woodwork/utils.py

```diff
@@ -54,6 +54,36 @@ def import_or_none(library):
         return None
 
 
+def ww_concat(base, to_add):
+    """Selects and calls the appropriate concat method based on the type on the base and to_add series
+
+    Args:
+        base: base Series/DataFrame
+        to_add: Series/DataFrame to be concatenated
+
+    Returns:
+        Series/DataFrame: result of concatenation
+    """
+    dd = import_or_none("dask.dataframe")
+    ps = import_or_none("pyspark.pandas")
+
+    if isinstance(base, (pd.Series, pd.DataFrame)):
+        concatenated_obj = pd.concat([base, to_add])
+    elif ww.accessor_utils._is_dask_dataframe(
+        base,
+    ) or ww.accessor_utils._is_dask_series(base):
+        concatenated_obj = dd.concat([base, to_add])
+    elif ww.accessor_utils._is_spark_dataframe(
+        base,
+    ) or ww.accessor_utils._is_spark_series(base):
+        concatenated_obj = ps.concat([base, to_add])
+    else:
+        raise TypeError(
+            f"When attempting concatenation, encountered unknown type of base: {base}",
```

Comment:
Why not include `to_add` in the error message? The unknown type could belong to that argument as well right?